drivers/gpio/gpio_mchp_xec.c

```diff
@@ -13,6 +13,10 @@
 
 #include "gpio_utils.h"
 
+#define XEC_GPIO_EDGE_DLY_COUNT		8
```

Comment:
@scottwcpg I think you can remove debug aids in the final patch

Reply:
@albertofloyd Where to find info about XEC_GPIO_EDGE_DLY_COUNT ?